tests/files/vasp/inputs/POSCAR_LiFePO4

```diff
@@ -1,31 +1,37 @@
-LiFePO4
--300.65685512
-    10.4117668700     0.0000000000     0.0000000000
-     0.0000000000     6.0671718800     0.0000000000
-     0.0000000000     0.0000000000     4.7594895400
-4 4 16
+Li4 Fe4 P4 O16
```

Comment:
Well...This is a huge pitfall, there is a POSCAR called `POSCAR_LiFePO4` and another POSCAR `LiFePO4.vasp` where the latter does not correspond to a `LiFePO4` structure at all

Reply:
I think it is FePO4. If you look at git blame and the commits it has been inteoduced to recognize the .vasp ending.

Reply:
Thus, nothing to worry about, imo.

Reply:
Yes looks like so. But I was trapped for a while trying to find out why tests are failing because I renamed it to `POSCAR_LiFePO4` (overwrote the other file) 😄

Reply:
Yes it was added in #1233, so I might get it back and correct its name

Reply:
Yeah, can imagine. Very confusing!